main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -358,6 +358,18 @@ object DocAst {
     def Var(id: Int): Type = AsIs(s"var$id")
   }
 
+  sealed trait Effect
```

Comment:
This should probably be called Eff

Reply:
or maybe even better Purity.

Reply:
It will later also capture things like `r1 + r2` so I pick Eff